pkg/providers/launchtemplate/launchtemplate.go

```diff
@@ -113,21 +111,22 @@ func (p *Provider) getStaticParameters(ctx context.Context, instanceType *cloudp
 	if err := instanceType.Requirements.Compatible(scheduling.NewRequirements(scheduling.NewRequirement(v1.LabelArchStable, v1.NodeSelectorOpIn, corev1beta1.ArchitectureArm64))); err == nil {
 		arch = corev1beta1.ArchitectureArm64
 	}
-	// TODO: make conditional on either Azure CNI Overlay or pod subnet
-	vnetLabels, err := p.getVnetInfoLabels(ctx, nodeClass)
-	if err != nil {
-		return nil, err
-	}
-	labels = lo.Assign(labels, vnetLabels)
 
-	// TODO: Make conditional on epbf dataplane
```

Comment:
https://github.com/Azure/karpenter-provider-azure/pull/318 this is addressed here, comment shouldnt have been removed yet though

Reply:
add the comment line back then?